src/Pages/HomePage.tsx

```diff
@@ -14,11 +16,17 @@ const HomePage = (): JSX.Element => {
     anchorEl,
     sidebarItems,
     selectedMenu,
+    userBMI,
+    percentUserBMI,
     sidebarItemClick,
     logOut
   } = useHomePage()
   useDefaultRedirect()
-  getUser()
+
+  useEffect(() => {
+    getUser()
+    window.history.replaceState({}, '', '/?menu=profile')
```

Comment:
why do we need this replace in `useEffect`, you can add this `searchParam` when you redirect to the `HomePage`, also let's change name of searchParam, `activeMenuItem`, smth like this or you can provide another one

Reply:
we need this to set searchParams by default when the application is launched